pom.xml

```diff
@@ -52,17 +40,17 @@
     <dependency>
       <groupId>org.apache.sshd</groupId>
       <artifactId>sshd-core</artifactId>
-      <version>0.14.0</version>
+      <version>1.2.0</version> <!-- TODO 1.3.0 requires Java 8 -->
```

Comment:
🐛  AFAIK it implies binary compatibility issues.
Before we go forward, we need to at least ensure it does not cause the collateral damage since the module's API is publicly exposed to the core and plugins